DF faid to start in Arch Linux (Antegros)

I have dwarffortress-lnp-git installed from AUR.

```
$pwd                                                                  
/opt/dwarffortress-lnp-git

$./startlnp                                                           
LNP/utilities/dwarf_therapist/DwarfTherapist (No such file or directory)
[WARNING] startlnp: Failed to start PyLNP

$cd df_linux

$./df                                                                                                                                                       
./libs/Dwarf_Fortress: /usr/lib/dwarffortress-lnp-git/df_linux/libs/libstdc++.so.6: version `CXXABI_1.3.5' not found (required by /usr/lib/dwarffortress-lnp-git/df_linux/libs/libgraphics.so)
./libs/Dwarf_Fortress: /usr/lib/dwarffortress-lnp-git/df_linux/libs/libstdc++.so.6: version `CXXABI_1.3.8' not found (required by /usr/lib/dwarffortress-lnp-git/df_linux/libs/libgraphics.so)
./libs/Dwarf_Fortress: /usr/lib/dwarffortress-lnp-git/df_linux/libs/libstdc++.so.6: version `GLIBCXX_3.4.18' not found (required by /usr/lib/dwarffortress-lnp-git/df_linux/libs/libgraphics.so)
./libs/Dwarf_Fortress: /usr/lib/dwarffortress-lnp-git/df_linux/libs/libstdc++.so.6: version `GLIBCXX_3.4.21' not found (required by /usr/lib/dwarffortress-lnp-git/df_linux/libs/libgraphics.so)
./libs/Dwarf_Fortress: /usr/lib/dwarffortress-lnp-git/df_linux/libs/libstdc++.so.6: version `GLIBCXX_3.4.15' not found (required by /usr/lib/dwarffortress-lnp-git/df_linux/libs/libgraphics.so)

$./distro_fixes.sh                      
[distro_fixes] [ERROR] DF_DIR not defined.

$pacman -Ss GLIBC
multilib/lib32-glibc 2.24-2 [installed]

$pacman -Ss libc++abi           
community/libc++abi 3.8.0-1 [installed]

```

--- a/pack/df_linux/distro_fixes.sh
+++ b/pack/df_linux/distro_fixes.sh
@@ -115,7 +115,7 @@ if [ x"$DF_ARCH" = x'32-bit' ] && [ x"$ARCH" = x'x86_64' ]; then
     # Gentoo 2.2
     elif [ x"$OS" = x'gentoo' ] || [ x"$OS" = x'sabayon' ]; then
         find_zlib /lib32/libz.so.1 /lib32
-    elif [ x"$OS" = x'arch' ] || [ x"$OS" = x'antergos' ] || [ x"$OS" = x'manjarolinux' ]; then
+    elif [ x"$OS" = x'arch' ] || [ x"$OS" = x'antergos' ] || [ x"$OS" = x'manjarolinux' ] || [ x"$OS" = x'chakra' ]; then
         find_zlib /usr/lib32/libz.so /usr/lib32
         if [ -e "/usr/lib32/libstdc++.so.6" ]; then
             export PRELOAD_LIB="${PRELOAD_LIB:+$PRELOAD_LIB:}/usr/lib32/libstdc++.so.6"
@@ -138,9 +138,8 @@ if [ x"$DF_ARCH" = x'32-bit' ] && [ x"$ARCH" = x'x86_64' ]; then
     if [ -z "$ZLIB_PATH" ]; then
         dlog WARN "Could not find a 32-bit zlib"
     fi
-
 elif [ x"$DF_ARCH" = x'64-bit' ] && [ x"$ARCH" = x'x86_64' ]; then
-    if [ x"$OS" = x'arch' ] || [ x"$OS" = x'antergos' ] || [ x"$OS" = x'manjarolinux' ]; then
+    if [ x"$OS" = x'arch' ] || [ x"$OS" = x'antergos' ] || [ x"$OS" = x'manjarolinux' ] || [ x"$OS" = x'chakra' ]; then
         find_zlib /usr/lib64/libz.so /usr/lib64
         if [ -e "/usr/lib64/libstdc++.so.6" ]; then
             export PRELOAD_LIB="${PRELOAD_LIB:+$PRELOAD_LIB:}/usr/lib64/libstdc++.so.6"
@@ -155,7 +154,6 @@ elif [ x"$DF_ARCH" = x'64-bit' ] && [ x"$ARCH" = x'x86_64' ]; then
     if [ -z "$ZLIB_PATH" ]; then
         dlog WARN "Could not find a 64-bit zlib"
     fi
-
 elif [ x"$DF_ARCH" = x'32-bit' ]; then
     find_zlib
 fi

--- a/pack/df_linux/distro_fixes.sh
+++ b/pack/df_linux/distro_fixes.sh
@@ -139,6 +139,23 @@ if [ x"$DF_ARCH" = x'32-bit' ] && [ x"$ARCH" = x'x86_64' ]; then
         dlog WARN "Could not find a 32-bit zlib"
     fi
 
+elif [ x"$DF_ARCH" = x'64-bit' ] && [ x"$ARCH" = x'x86_64' ]; then
+    if [ x"$OS" = x'arch' ] || [ x"$OS" = x'antergos' ] || [ x"$OS" = x'manjarolinux' ]; then
+        find_zlib /usr/lib64/libz.so /usr/lib64
+        if [ -e "/usr/lib64/libstdc++.so.6" ]; then
+            export PRELOAD_LIB="${PRELOAD_LIB:+$PRELOAD_LIB:}/usr/lib64/libstdc++.so.6"
+        else
+            dlog WARN "Could not find /usr/lib64/libstdc++.so.6"
+        fi
+    else
+        dlog "WARN" "64bit 'Dwarf_Fortress' on unhandled 64bit OS detected. If you get 'missing file' errors, please open an issue on Github: https://github.com/Lazy-Newb-Pack/Lazy-Newb-Pack-Linux/issues."
+        find_zlib
+    fi
+
+    if [ -z "$ZLIB_PATH" ]; then
+        dlog WARN "Could not find a 64-bit zlib"
+    fi
+
 elif [ x"$DF_ARCH" = x'32-bit' ]; then
     find_zlib
 fi

